charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 4.3.1
+version: 4.3.2
```

Comment:
Imho minor version should be changed as new feature is added.

So right version would be version 4.4.0.

Reply:
Changed and resolved in https://github.com/prometheus-community/helm-charts/pull/42/commits/27e8ee1f9e4ce2320e3a29f55f3e052a95da5075